### ISSUE TITLE
hla.dat File sequence version changes

### ISSUE THREAD

**Issue by jlimauro** [ANHIG/IMGTHLA#370]
In 3.56.0 dat file update, I noticed that for many alleles the sequence version for allele has changed, for example, HLA00001 had an SV of 1 now its 4. What explains this difference? Fixes missed in previous versions?

Additionally for alleles in which have been delete will the file now show an '0' instead of 'x' on the line following SQ?

Just looking for clarity in these changes and if they are one off or what should be expected in future releases.

**Comment by jrob119** [comment]
The SV lines have been reviewed and updated to better represent sequence changes over the various releases. This change has been made to the hla.dat files in all previous releases.

To ensure all hla.dat files are consistently formatted we have also updated a number of additional lines in the oldest files. No sequences or allele designations were changed.

To ensure compatibility with third party data parsers and the EBML-ENA guidelines for flat files, any deleted alleles were updated to have a blank sequence with a 0, representing the sequence length rather than an X.

These conventions will be continued in further releases.
